docs/logging/README.md

```diff
@@ -99,7 +102,9 @@ NOTE:
     "X-Xss-Protection": "1; mode=block"
   },
   "tags": {
-    "objectsAndPools": {
+    "objectErasureMap": {
+      "pool": 1,
```

Comment:
Actually we should include this information per object, in case of multi-delete, each object can be in a different pool/set